pureport_client/__main__.py

```diff
@@ -46,130 +50,40 @@ def cli(ctx, api_url, api_key, api_secret, api_profile, access_token):
-                    'commands': find_client_commands(Client.AccountNetworksClient)
-                },
-                {
-                    'context': Client.AccountsClient.permissions,
-                    'commands': find_client_commands(Client.AccountPermissionsClient)
-                },
-                {
-                    'context': Client.AccountsClient.ports,
-                    'commands': find_client_commands(Client.AccountPortsClient)
-                },
-                {
-                    'context': Client.AccountsClient.roles,
-                    'commands': find_client_commands(Client.AccountRolesClient)
-                },
-                {
-                    'context': Client.AccountsClient.supported_connections,
-                    'commands': find_client_commands(Client.AccountSupportedConnectionsClient)
-                },
-                {
-                    'context': Client.AccountsClient.supported_ports,
-                    'commands': find_client_commands(Client.AccountSupportedPortsClient)
-                }
-            ]
-        },
-        {
-            'context': Client.cloud_regions,
-            'commands': find_client_commands(Client.CloudRegionsClient)
-        },
-        {
-            'context': Client.cloud_services,
-            'commands': find_client_commands(Client.CloudServicesClient)
-        },
-        {
-            'context': Client.connections,
-            'commands': find_client_commands(Client.ConnectionsClient)
-        },
-        {
-            'context': Client.facilities,
-            'commands': find_client_commands(Client.FacilitiesClient)
-        },
-        {
-            'context': Client.gateways,
-            'commands': find_client_commands(Client.GatewaysClient)
-        },
-        {
-            'context': Client.locations,
-            'commands': find_client_commands(Client.LocationsClient)
-        },
-        {
-            'context': Client.networks,
-            'commands': [
-                Client.NetworksClient.get,
-                Client.NetworksClient.update,
-                Client.NetworksClient.delete,
-                {
-                    'context': Client.NetworksClient.connections,
-                    'commands': find_client_commands(Client.NetworkConnectionsClient)
-                }
-            ]
-        },
-        {
-            'context': Client.options,
-            'commands': find_client_commands(Client.OptionsClient)
-        },
-        {
-            'context': Client.ports,
-            'commands': find_client_commands(Client.PortsClient)
-        },
-        {
-            'context': Client.supported_connections,
-            'commands': find_client_commands(Client.SupportedConnectionsClient)
-        },
-        {
-            'context': Client.tasks,
-            'commands': find_client_commands(Client.TasksClient)
-        }
-    ]
+    # XXX: this function will dynamically discover the command tree based on
```

Comment:
`XXX`

Is this the same as:
```
   _                   _
 _( )                 ( )_
(_, |      __ __      | ,_)
   \'\    /  ^  \    /'/
    '\'\,/\      \,/'/'
      '\| []   [] |/'
        (_  /^\  _)
          \  ~  /
          /HHHHH\
        /'/{^^^}\'\
    _,/'/'  ^^^  '\'\,_
   (_, |           | ,_)
     (_)           (_)
```

Reply:
@kevinkirkup https://stackoverflow.com/questions/1452934/what-is-the-meaning-of-xxx-in-code-comments#:~:text=Basically%20the%20first%20two%20(%20NOTE,and%20TODO%20)%20do%20require%20action.